src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -334,6 +335,7 @@ public static SlimefunItem getByItem(ItemStack item) {
 			else if (sfi instanceof DamagableChargableItem && SlimefunManager.isItemSimiliar(item, sfi.getItem(), false)) return sfi;
 			else if (sfi instanceof ChargedItem && SlimefunManager.isItemSimiliar(item, sfi.getItem(), false)) return sfi;
 			else if (sfi instanceof SlimefunBackpack && SlimefunManager.isItemSimiliar(item, sfi.getItem(), false)) return sfi;
+			else if ((SlimefunManager.isItemSimiliar(item, SlimefunItems.BROKEN_SPAWNER, false)) || (SlimefunManager.isItemSimiliar(item, SlimefunItems.REPAIRED_SPAWNER, false))) return sfi;
```

Comment:
This method performs a for loop, you should place this special case outside of the for loop, otherwise it will perform the same check a thousand times per ItemStack, we only need it once though.